src/wallet.js

```diff
@@ -618,6 +618,12 @@ Wallet.prototype.transactions = function(params, callback) {
     }
     args.push('minHeight=' + params.minHeight);
   }
+  if (params.compact) {
```

Comment:
Hm, I don't think this is going to work if the user sets compact to false. The logic should probably be: if (typeof (params.compact) !== 'undefined') instead

Reply:
actually the default behavior now is replying with not compact transactions, so it actually worked when I tried it, providing `false` just makes it use defaults. But will push update for the sake of completeness.